CHANGELOG.md

```diff
@@ -24,6 +24,24 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+### [shopsys/framework]
+#### Changed
+- [#271 - Complete refactoring of feeds functionality](https://github.com/shopsys/shopsys/pull/271)
+  - modules are responsible for querying the data to improve performance
+  - interfaces from package product-feed-interface are not used anymore as they were only important with open-box architecture
+  - only relevant data is fetched from the database, should result in enhanced performance
```

Comment:
I though that it really helped the performance so why "should"? :slightly_smiling_face:

Reply:
Because I do not have the exact number for all the feeds. I consider it a hunch at this point. But some preliminary test of Google feed shown about eight-fold increase in speed.

Reply:
ok, fair enough